src/platform.ts

```diff
@@ -69,14 +71,17 @@ export class LinuxDistribution {
                 else if (key === 'VERSION_ID') {
                     version = value;
                 }
+                else if (key === 'ID_LIKE') {
+                    idLike = value.split(" ");
+                }
 
                 if (name !== unknown && version !== unknown) {
```

Comment:
Is that what it does though? It looks like it will bail if `ID_LIKE` hasn't been encountered and we find `ID` and `VERSION_ID` values -- even if we don't understand them.